runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/modeler-palette/modeler-palette.scss

```diff
@@ -0,0 +1,13 @@
+.djs-palette .entry {
+    cursor: pointer;
+    transition: 0.15s background-color;
+
+    &:hover {
+        color: initial;
+        background-color: #f1f1f3;
```

Comment:
try to substitute hex with predefined colour from the palette, see https://mui.com/material-ui/customization/default-theme/